### PR TITLE
remove per-worker timings

### PR DESCRIPTION
**Pull request by blampe**
Per-worker latency timings represent a majority of our metrics but rolling these up to the host level is sufficient.

@kriskowal @sloppyfocus

### CODE REVIEW

**Comment by kriskowal** [comment]
@Raynos please sign. CI will pass soon.

**Comment by Raynos** [comment]
This is wrong location for this fix. Look in batch-stats instead.

https://github.com/uber/hyperbahn/blob/master/clients/batch-stats.js

**Comment by kriskowal** [comment]
@Raynos i.e., axe this line? https://github.com/uber/hyperbahn/blob/master/clients/batch-stats.js#L67

**Comment by blampe** [comment]
@kriskowal I'll let you take it from here :)

File: clients/dual-statsd.js [hunk 1]
```diff
@@ -114,7 +114,6 @@ DualStatsd.prototype.timing = function timing(name, time) {
 
     self.globalClient.timing(name, time);
     self.perServerClient.timing(name, time);
-    self.perWorkerClient.timing(name, time);
```
Comment (Raynos) [inline clients/dual-statsd.js]: We want to keep this line.

This lets us track event loop lag per worker.

File: clients/batch-stats.js [hunk 2]
```diff
@@ -64,7 +64,6 @@ function handleStat(stat) {
         stat.name === 'tchannel.inbound.calls.recvd'
     ) {
         self.writeToStatsd(self.statsd.globalClient, stat);
-        self.writeToStatsd(self.statsd.perWorkerClient, stat);
```
Comment (Raynos) [inline clients/batch-stats.js]: We want to keep this line.

I recommend we keep `recvd` per worker as it allows us to understand what QPS is going through a single HOT worker. this is a common debugging technique based on our alerts.

I recommend we remove `latency` for most service with the exception of the `hyperbahn` service being whitelisted, tracking the hyperbahn latency on a per-worker basis allows us to debug any issues with advertise.

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**